<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUi</class>
 <widget class="QMainWindow" name="MainWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GDB ImageWatch</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame { border: 0; }
QFrame#frame_2 { margin-top: 5px; }
QStatusBar::item { border: none; };</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="SymbolSearchInput" name="symbolList">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
            <property name="placeholderText">
             <string>add symbols (ctrl+k)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="imageList">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="rightPanelLayout">
          <item>
           <widget class="QWidget" name="toolbar" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <widget class="QToolButton" name="acToggle">
             <property name="geometry">
              <rect>
               <x>50</x>
               <y>0</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Toggle Auto Contrast</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="acEdit">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>0</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Toggle min and max intensity editor</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="reposition_buffer">
             <property name="geometry">
              <rect>
               <x>90</x>
               <y>0</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string extracomment="Reposition buffer on center">Reposition buffer to fit window</string>
             </property>
            </widget>
            <widget class="QToolButton" name="linkViewsToggle">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>0</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Move all buffers simultaneously</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="rotate_90_cw">
             <property name="geometry">
              <rect>
               <x>170</x>
               <y>0</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Rotate 90° clockwise</string>
             </property>
            </widget>
            <widget class="QToolButton" name="rotate_90_ccw">
             <property name="geometry">
              <rect>
               <x>210</x>
               <y>0</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Rotate 90° counterclockwise</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="minMaxEditor" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>85</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>85</height>
             </size>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>540</width>
               <height>84</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>10</number>
              </property>
              <property name="verticalSpacing">
               <number>0</number>
              </property>
              <item row="0" column="1">
               <widget class="QLabel" name="label_red_min">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Minimum value for red (first) channel. Smaller values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
               <widget class="QLabel" name="label_min">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>38</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLineEdit" name="ac_red_min">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_green_min">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Minimum value for green (second) channel. Smaller values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLineEdit" name="ac_green_min">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="label_blue_min">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Minimum value for blue (third) channel. Smaller values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QLineEdit" name="ac_blue_min">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignTop">
               <widget class="QLabel" name="label_max">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>38</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_red_max">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Maximum value for red (first) channel. Larger values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="ac_red_max">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_green_max">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Maximum value for green (second) channel. Larger values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QLineEdit" name="ac_blue_max">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLineEdit" name="ac_green_max">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="9">
               <widget class="QToolButton" name="ac_reset_max">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Reset maximum auto contrast levels to the largest values found in the buffer.</string>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="label_blue_max">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Maximum value for blue (third) channel. Larger values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="9">
               <widget class="QToolButton" name="ac_reset_min">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Reset minimum auto contrast levels to the lowest values found in the buffer.</string>
                </property>
               </widget>
              </item>
              <item row="1" column="7">
               <widget class="QLabel" name="label_alpha_max">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Maximum value for alpha (fourth) channel. Larger values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QLabel" name="label_alpha_min">
                <property name="minimumSize">
                 <size>
                  <width>20</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="toolTip">
                 <string>Minimum value for alpha (fourth) channel. Smaller values will be clamped at this level when auto contrast is enabled.</string>
                </property>
               </widget>
              </item>
              <item row="0" column="8">
               <widget class="QLineEdit" name="ac_alpha_min">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item row="1" column="8">
               <widget class="QLineEdit" name="ac_alpha_max">
                <property name="font">
                 <font>
                  <pointsize>10</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="GLCanvas" name="bufferPreview" native="true">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLCanvas</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/gl_canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SymbolSearchInput</class>
   <extends>QLineEdit</extends>
   <header>src/ui/symbol_search_input.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>acEdit</sender>
   <signal>toggled(bool)</signal>
   <receiver>minMaxEditor</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
